main/keychain/src/main/java/org/cryptomator/keychain/LinuxSystemKeychainAccess.java

```diff
@@ -23,6 +23,7 @@ public LinuxSystemKeychainAccess() {
 	}
 
 	private static Optional<KeychainAccessStrategy> constructKeychainAccess() {
+		if (!SystemUtils.IS_OS_LINUX) return Optional.empty();
```

Comment:
Always use full "block" style, even for short statements

```suggestion
		if (!SystemUtils.IS_OS_LINUX) {
			return Optional.empty();
		}
```